Changelog.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 ### Konekt Enum Eloquent
 
+## 1.10.0
+##### 2023-08-03
```

Comment:
Thanks for taking care of this as well :ok_hand: Since we don't know the release number/date in advance, please just leave it as [unreleased](https://keepachangelog.com/en/1.0.0/#effort), like this:

```suggestion
## Unreleased
##### 2023-XX-YY
```